src/eventfd.rs

```diff
@@ -5,21 +5,36 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE-BSD-3-Clause file.
 
+//! Structure and wrapper functions for working with
+//! [`eventfd`](http://man7.org/linux/man-pages/man2/eventfd.2.html).
+
 use std::fs::File;
 use std::os::unix::io::{AsRawFd, FromRawFd, RawFd};
 use std::{io, mem, result};
 
 use libc::{c_void, dup, eventfd, read, write};
 
-/// A safe wrapper around a Linux eventfd (man 2 eventfd).
+/// A safe wrapper around Linux
+/// [`eventfd`](http://man7.org/linux/man-pages/man2/eventfd.2.html).
 pub struct EventFd {
     eventfd: File,
 }
 
 impl EventFd {
-    /// Creates a new blocking EventFd with an initial value.
+    /// Create a new blocking EventFd with an initial value.
```

Comment:
The eventfd is not necessarily blocking as you can pass flags in which you can specify `EFD_NONBLOCK`. We can just remove the word "blocking" for the documentation to be accurate.